be/src/exec/hdfs_scanner_orc.cpp

```diff
@@ -18,6 +18,8 @@
 
 #include "exec/exec_node.h"
 #include "exec/iceberg/iceberg_delete_builder.h"
+#include "exprs/base64.h"
+#include "formats/orc/apache-orc/c++/src/security/InMemoryKeystore.hh"
```

Comment:
We should put all the ORC-related header files which are exposed to users in `apache-orc/c++/include/orc`